### DISCUSSION TITLE
Guide on using for Qwik SPA mode

### DISCUSSION THREAD

**Discussion by ahnpnl** [robisim74/qwik-speak#91] [Q&A]
Hi,

In my app, there are a few pages I want to use client side navigation via hook `useNavigate`. I’m using `Speak` component to load translations for those pages. 

If I use full page refresh navigation, the translations can load properly. However, if I use client side navigation, the translations are not loaded. Is there a way to make this library work with that navigation mode? A guide in docs would be really appreciated 🙏

**Comment by robisim74** [comment]
> if I use client side navigation, the translations are not loaded

It sounds weird to me. The application in this repo also uses client side navigation (SPA mode), and it works correctly.

Could you provide a reproduction?

**Comment by robisim74** [comment]
I'm closing due to lack of reproduction. If you still have problems, please open an issue and provide a repro.
